ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -334,6 +334,17 @@ objects:
             FILTER=gzip queue /tmp/artifacts/pods/${file}_previous.log.gz oc logs --request-timeout=20s -p $i
           done < /tmp/containers
 
+          mkdir -p /tmp/artifacts/images
+          for FULLY_QUALIFIED_IMAGE in $(jq -r '.items[].status.containerStatuses[].imageID' /tmp/artifacts/pods.json | sort | uniq); do
+            NAMESPACED_IMAGE="${FULLY_QUALIFIED_IMAGE#docker-pullable*//registry.svc.ci.openshift.org/}"
+            if [[ "${NAMESPACED_IMAGE}" = "${FULLY_QUALIFIED_IMAGE}" ]]; then
+              continue
+            fi
+            NAMESPACE="${NAMESPACED_IMAGE%/*}"
+            IMAGE="${NAMESPACED_IMAGE#*/}"
+            KUBECONFIG=FIXME queue "/tmp/artifacts/images/${IMAGE}" oc -s https://api.ci.openshift.org --request-timeout=5s get imagestreamimage -n "${NAMESPACE}" "${IMAGE}" -o json
```

Comment:
@smarterclayton, @stevekuznetsov: is there a kubeconfig I could use here (instead of the `FIXME`) to reach back into `api.ci.openshift.org` for the image metadata?

Reply:
This is something we should be doing in the release manifest.  We already capture the release payload when we build it https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/533/pull-ci-openshift-installer-master-e2e-aws/921/artifacts/release-latest/release-payload/image-references

Once we settle on the image annotations, I'm going to update image-references to contain them.  At that point you don't need anything else.

Reply:
> This is something we should be doing in the release manifest.

But not all images are from the release, right?  Ideally we'd have images throughout the ecosystem setting these annotations, so we could get the repo, commit, etc. for `quay.io/coreos/etcd@sha256:5b6691b7225a3f77a5a919a81261bbfb31283804418e187f7116a0a9ef65d21d` and other third-party images consumed by the cluster.

Reply:
There are no third party images consumed by the cluster during a normal run (everything has to be first party to be shipped).

If you're trying to get at the metadata of the image in use, this snippet can only grab from images that are in the release payload (those are the only ones that will be in api.ci) which is why I think the intent (get metadata about running images) can be accomplished by us fixing the release payload.  The only imagestreamimage objects you will be able to retrieve will be those that were part of the payload.